src/newrelic/monitor_script.js

```diff
@@ -52,6 +52,7 @@ $http.get('$$$URL$$$',
           return;
         }
         try {
+          details.request = {};
```

Comment:
```suggestion
          delete details.request;
```

Reply:
isn't this a be aggressive? it would suffice to delete the `Authorization` header, I think.

Reply:
```js
   delete details.request;
```
I tried that first, but it threw the following error during execution in New Relic:
```
TypeError: Cannot read property 'uri' of undefined
    at requestToJSON (/opt/runtimes/4.0.0/node_modules/request/request.js:76:15)
    at IncomingMessage.responseToJSON [as toJSON] (/opt/runtimes/4.0.0/node_modules/request/request.js:89:28)
    at JSON.stringify (<anonymous>)
    at Request.$http.get [as _callback] (eval at JobResource.getScriptFn (/opt/runtimes/4.0.0/modules/synthetics-runner/lib/job-resource/index.js:78:19), <anonymous>:52:52)
    at Request.self.callback (/opt/runtimes/4.0.0/node_modules/request/request.js:185:22)
    at Request.emit (events.js:182:13)
    at Request.<anonymous> (/opt/runtimes/4.0.0/node_modules/request/request.js:1161:10)
    at Request.emit (events.js:182:13)
    at IncomingMessage.<anonymous> (/opt/runtimes/4.0.0/node_modules/request/request.js:1083:12)
    at Object.onceWrapper (events.js:273:13)
```
Even when I copied the `details` object using `Object.assign()`...

Reply:
> isn't this a be aggressive?

Well, there is nothing in this object that is of real interest to us. It stems from the request we explicitly fire to get the activation details after checking the action's status, and it just contains stuff we already know: the URL, the method and 2 headers: `content-type` and `authorization`. I think it's just a waste of space.

Reply:
ok.